### PR TITLE
Fix:Alert Controller in logout cell in SettingVC

### LINKED ISSUE #500: Add Alert Controller in logout cell in SettingsVC
**Actual Behaviour**

No Alert Controller in logout cell in SettingsVC

**Expected Behaviour**

Should be present , as similar to Android App

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by baquer**
Fixes #500 

Screenshots for the change: 

screenshot 2019-01-21 at 12 36 38 am

### CODE REVIEW

**Comment by baquer** [comment]
@chashmeetsingh , @jogendra , Please review the PR.

File: Susi/Controllers/SettingsController/SettingsViewController.swift [hunk 1]
```diff
@@ -125,8 +125,19 @@ class SettingsViewController: UITableViewController {
             } else if row == 2 {
                 presentResetPasswordController()
             } else if row == 3 {
-                logoutUser()
-            } 
+                if logoutTitle.text == ControllerConstants.Settings.logout.localized() {
+                    let logoutAlert = UIAlertController(title: ControllerConstants.Logout.title, message: ControllerConstants.Logout.message, preferredStyle: UIAlertController.Style.alert)
+                    logoutAlert.addAction(UIAlertAction(title: ControllerConstants.Logout.cancel, style: .default, handler: { (action: UIAlertAction!) in
+                        logoutAlert .dismiss(animated: true, completion: nil)
```
Comment (jogendra) [inline Susi/Controllers/SettingsController/SettingsViewController.swift]: Why there is space between `logoutAlert` and `dismiss`

File: Susi/Helpers/ControllerConstants.swift [hunk 2]
```diff
@@ -273,5 +273,12 @@ class ControllerConstants {
         static let reportIssueURL = "https://github.com/fossasia/susi_iOS/issues"
         static let licenseURL = "https://github.com/fossasia/susi_iOS/blob/master/LICENSE"
     }
+    
+    struct Logout {
+        static let title = "Logout"
+        static let message = "Are you sure , you want to logout?"
```
Comment (jogendra) [inline Susi/Helpers/ControllerConstants.swift]: It is unprofessional to keep space before a comma. so change it to `Are you sure, you want to log out?`

Comment (baquer) [reply Susi/Helpers/ControllerConstants.swift]: changes done , please review

File: Susi/Controllers/SettingsController/SettingsViewController.swift (hunk 1 above)
Comment (baquer) [reply Susi/Controllers/SettingsController/SettingsViewController.swift]: changes done , please review

**Comment by jogendra** [comment]
@samagragupta Please don't make an approval on every approved PR without reviewing the code. This doesn't take you (benefits you) anywhere. You know you will be blocked from the Org. We had a lot of people like you, please stop this.

**Pull request merged**